frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -383,7 +382,14 @@ export const surveyLogic = kea<surveyLogicType>([
                 const responseJSON = await api.query(query)
                 const { results } = responseJSON
 
-                const events = results?.map((r) => ({ properties: JSON.parse(r[0]), distinct_id: r[1] }))
+                const events =
+                    results?.map((r) => {
+                        const distinct_id = r[0]
+                        const properties = JSON.parse(r[1])
+                        const personProperties = JSON.parse(r[2])
+                        properties.email = personProperties.email
```

Comment:
this is too hacky and wouldn't work in cases where, for example, the team uses a different property to identify people.

This is the code that gets called in <PersonDisplay> -> https://github.com/PostHog/posthog/blob/master/frontend/src/scenes/persons/person-utils.ts#L36

So, we'll want to pass in the entire person object, will all person properties here. So, return all three here and use appropriately.

This is the type to pass into <PersonDisplay> -> https://github.com/PostHog/posthog/blob/master/frontend/src/scenes/persons/PersonDisplay.tsx#L11